<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>991</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(195, 190, 180);
border-radius:30px
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,210);
</string>
      </property>
      <widget class="QWidget" name="horizontalWidget_3" native="true">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>0</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(255,255,255,255)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PEDIDO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalWidget_4" native="true">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>0</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCerrar">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnGuardar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);
border-radius:30px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblPedido">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>411</width>
        <height>371</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border: 3px solid  #000000;
background-color: rgb(199, 194, 184);
</string>
      </property>
      <column>
       <property name="text">
        <string>Nro.Pedido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Provedor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comp. Pago</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnCancelar">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
border:1.5px solid #7c6e59;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Marohuma_icono.png);
background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes/QT_imagenes.qrc">:/newPrefix/Marohuma_icono.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Nro. Pedido:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNroDocumento">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
border:1.5px solid #7c6e59;
border-radius:15px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <property name="placeholderText">
       <string>Ingrese el numero del Pedido</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dtFecha">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:10px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>Provedor:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboProveedor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:10px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <item>
       <property name="text">
        <string>--Seleccionar--</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lbID">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>widget_2</zorder>
     <zorder>tblPedido</zorder>
     <zorder>label</zorder>
     <zorder>label_11</zorder>
     <zorder>txtNroDocumento</zorder>
     <zorder>label_12</zorder>
     <zorder>dtFecha</zorder>
     <zorder>label_15</zorder>
     <zorder>cboProveedor</zorder>
     <zorder>btnGuardar</zorder>
     <zorder>btnCancelar</zorder>
     <zorder>lbID</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes/QT_imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
